samples/matter/lock/README.rst

```diff
@@ -72,6 +72,28 @@ Configuration
 
 |config|
 
+.. matter_door_lock_sample_configuration_file_types_start
+
+The sample uses different configuration files depending on the supported features.
+The configuration files are automatically attached to build based on the filename build type suffix like for example ``_dfu`` in ``prj_dfu.conf`` file.
+In case of modifying configuration options, you should apply it to the files with appropriate suffix in order to achieve desired effect.
+See the table below for information about available configuration types:
+
++------------------+--------------------------------------+------------------------+------------------------------+
+| Config suffix    | Enabled feature                      | Enabling build option  | Supported boards             |
++==================+======================================+========================+==============================+
+| none             | none (basic build)                   | none                   | ``nrf52840dk_nrf52840``      |
+|                  |                                      |                        | ``nrf5340dk_nrf5340_cpuapp`` |
+|                  |                                      |                        | ``nrf21540dk_nrf52840``      |
++------------------+--------------------------------------+------------------------+------------------------------+
+| _dfu             | Single-image Device Firmware Upgrade | ``-DBUILD_WITH_DFU=1`` | ``nrf52840dk_nrf52840``      |
+|                  |                                      |                        | ``nrf21540dk_nrf52840``      |
```

Comment:
This is not supported.

Reply:
Right, I suggested by looking at nrf21540dk_nrf52840_dfu.conf file name. But I think this file is not used anywhere, so I will remove it.